docs/HACKING.md

```diff
@@ -55,7 +59,10 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
 
     $ mkvirtualenv -a $PROJECT_HOME/goldstone-server goldstone-server
 
-   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original.
+   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original. Then:
+
+    $ workon goldstone-server
+    $ pip install tox
```

Comment:
The commands to pip install -r requirements.txt and test-requirements.txt need to be added here. Without them, a developer can't interactively debug Goldstone code.